src/partials/hero.html

```diff
@@ -1,75 +1,282 @@
+            <stop
+              offset="0.411317"
+              stop-color="white"
+              stop-opacity="0.441769"
+            />
+            <stop offset="1" stop-color="white" stop-opacity="0" />
+          </linearGradient>
+        </defs>
+      </svg>
+      <svg
+        class="vector-tablet-mobile"
+        viewBox="0 0 486 742"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <mask id="path-1-inside-1_5503_36" fill="white">
+          <path
+            fill-rule="evenodd"
+            clip-rule="evenodd"
+            d="M485 499V243C485 109.347 376.653 1 243 1C109.347 1 1 109.347 1 243V499C1 632.653 109.347 741 243 741C376.653 741 485 632.653 485 499ZM243 0C108.795 0 0 108.795 0 243V499C0 633.205 108.795 742 243 742C377.205 742 486 633.205 486 499V243C486 108.795 377.205 0 243 0Z"
+          />
+        </mask>
+        <path
+          d="M486 499V243H484V499H486ZM486 243C486 108.795 377.205 0 243 0V2C376.101 2 484 109.899 484 243H486ZM243 0C108.795 0 0 108.795 0 243H2C2 109.899 109.899 2 243 2V0ZM0 243V499H2V243H0ZM0 499C0 633.205 108.795 742 243 742V740C109.899 740 2 632.101 2 499H0ZM243 742C377.205 742 486 633.205 486 499H484C484 632.101 376.101 740 243 740V742ZM243 -1C108.243 -1 -1 108.243 -1 243H1C1 109.347 109.347 1 243 1V-1ZM-1 243V499H1V243H-1ZM-1 499C-1 633.757 108.243 743 243 743V741C109.347 741 1 632.653 1 499H-1ZM243 743C377.757 743 487 633.757 487 499H485C485 632.653 376.653 741 243 741V743ZM487 499V243H485V499H487ZM487 243C487 108.243 377.757 -1 243 -1V1C376.653 1 485 109.347 485 243H487Z"
+          fill="url(#paint0_linear_5503_36)"
+          mask="url(#path-1-inside-1_5503_36)"
+        />
+        <defs>
+          <linearGradient
+            id="paint0_linear_5503_36"
+            x1="258"
+            y1="755"
+            x2="257"
+            y2="171"
+            gradientUnits="userSpaceOnUse"
+          >
+            <stop offset="0" stop-color="white" />
+            <stop
+              offset="0.411317"
+              stop-color="white"
+              stop-opacity="0.441769"
+            />
+            <stop offset="1" stop-color="white" stop-opacity="0" />
+          </linearGradient>
+        </defs>
+      </svg>
+    </div>
+    <div class="hero-wrapper">
+      <div class="wrapper-left-block">
+        <div class="left-block-description">
+          <p class="description">
+            Our expertly curated fruit baskets are made with the freshest,
+            highest quality fruits available. Whether you are looking for a
+            healthy snack or a gift for a loved one, Fresh Harvest Box has got
+            you covered.
+          </p>
+        </div>
+        <div class="left-block-card">
+          <div class="hero-card-watermelon hero-card">
+            <h2 class="visual-hidden">hero</h2>
+            <h3 class="hero-card-title title-card-watermelon">Fruit</h3>
+            <picture class="hero-card-img">
```

Comment:
zamiast picture zrobiłbym chyba jak w projekcie przynajmniej u mnie:

img
              class="hero-card-img"
              src="./img/hero/watermelon.png"
              srcset="
                ../imgwatermelon.png 1x,
                ../imgwatermelon@2x.png 2x
              "
              width="200"
              height="172"
              alt="watermelon"
            

Ale nie wiem czy tak by bylo ok. I zdjęcie wtedy dodaje się z ovalbym cieniem?

Reply:
aby dodać dwie retiny responsywnie czyli x1 i x2 dla mobile oraz x1 i x2 dla tablet oraz x1 i x2 dla desktop robi się to przez picture. tak mnie uczył Ivan. tak robiliśmy razem w ostatniej szóstej pracy więc napewno jest dobrze.
ten cień to svg dodane inline